docs/pages/desktop-access/getting-started.mdx

```diff
@@ -39,6 +39,8 @@ Teleport Desktop Access is not yet compatible with Windows Server 2022.
 
 - A Linux server to run the Teleport Desktop Access service on.
   You can reuse an existing server running any other Teleport instance.
+- A server or virtual machine running a Windows operating system with
+Remote Desktop Services (RDP) enabled and the RDP port available to the Linux server.
```

Comment:
Remote Desktop Services generally refers to Windows Server. Let's just say _with Remote Desktop enabled_.

Reply:
updated